src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/StructuredData.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { string, number } from 'prop-types';
+
+const StructuredData = ({
+  promoBrandTitle,
+  shortSynopsis,
+  durationISO8601,
+  embedUrl,
+  imageUrl,
+  episodeAvailableFrom,
+}) => {
+  const metadata = {
+    '@context': 'http://schema.org',
+    '@type': 'AudioObject',
+    name: promoBrandTitle,
+    description: shortSynopsis,
+    duration: durationISO8601,
+    thumbnailUrl: imageUrl,
+    uploadDate: new Date(episodeAvailableFrom).toISOString(),
```

Comment:
I think we should be using the `releaseDateTimeStamp` instead of `episodeAvailableFrom` as the upload date